### PR TITLE
[oneDNN] Identify new ISA

### PR DESCRIPTION
**Pull request by gaurides**
This PR adds code to identify new instruction sets

### CODE REVIEW

**Review by ezhulenev — APPROVED** [review]

**Comment by gaurides** [comment]
Hi @ezhulenev - Can you please share the failures in internal failures in feedback/copybara?

File: third_party/xla/third_party/tsl/tsl/platform/cpu_info.h [hunk 1]
```diff
@@ -132,6 +132,11 @@ enum CPUFeature {
   AMX_TILE = 41,  // Tile configuration and load/store
   AMX_INT8 = 42,  // Int8 tile matrix multiplication
   AMX_BF16 = 43,  // Bfloat16 tile matrix multiplication
+
+  AVX512_FP16 = 44,  // Float16 neural network
+  AMX_FP16 = 45,  // Float16 tile matrix multiplication
+  AVX_NE_CONVERT = 46, // Instructions for faster bfloat16, float16 convert.
```
Comment (cantonios) [inline third_party/xla/third_party/tsl/tsl/platform/cpu_info.h:138]: nit: these next two are missing a second space before the comment.

**Review by cantonios — CHANGES_REQUESTED** [review]

**Review by cantonios — APPROVED** [review]

**Comment by cantonios** [comment]
Hmm... now that XLA/TSL has been separated out of TF, it looks like we're having issues because this is a cross-cutting change.

I think you need to create a separate PR on the XLA repo to add the TSL definitions.  Then once that's merged, remove that part from this PR and submit only the TF-only changes.

@jakeharmon8 is that correct?

**Comment by gaurides** [comment]
> Hmm... now that XLA/TSL has been separated out of TF, it looks like we're having issues because this is a cross-cutting change.
> 
> I think you need to create a separate PR on the XLA repo to add the TSL definitions. Then once that's merged, remove that part from this PR and submit only the TF-only changes.
> 
> @jakeharmon8 is that correct?

ok, i will create XLA PR for those 2 files.

**Comment by gaurides** [comment]
@cantonios - this is the openXLA PR - This PR is for ISA related changes in XLA part. Earlier they were part of this PR - https://github.com/openxla/xla/pull/7565

**Comment by gaurides** [comment]
@cantonios - please let me know if I need to update/fix something for this PR. Thanks.

**Review by cantonios — APPROVED** [review]

**Pull request merged**